packages/fxa-settings/README.md

```diff
@@ -57,6 +58,54 @@ This application uses [Apollo client cache](https://www.apollographql.com/docs/r
 
 Access the client cache data in top-level objects via custom `use` hooks. At the time of writing, `useAccount` and `useSession` (see the "Sessions" section) will allow you to access `data.account` and `data.session` respectively inside components where that data is needed. See the "Testing" section of this doc for how to mock calls.
 
+### GQL error handling
+
+When an error occurs from a query or mutation, the Apollo Client GQL response distinguishes between two kinds of errors on the `error` object - `graphQLErrors`, an array of errors provided by a GQL server-side resolver, and `networkError`, an error that is thrown outside of a resolver. We don't need to send `graphQLErrors` to Sentry because they're logged by the `fxa-gql-api` package, but a `networkError` should be logged because it means the query was rejected and no data was returned, such as if Apollo Client failed to connect to the GQL endpoint.
+
+All errors should be handled. As a general rule of thumb, validation errors (which are in the `graphQLErrors` array) should be displayed the user, while all others should be reported to Sentry unless they are know to occur and should be intentionally ignored. Because GQL errors are essentially validation errors, broadly speaking only mutations that take an input need to be checked for GQL errors.
```

Comment:
I apologize in advance for being so picky here (and for not suggesting this originally, I did some reading this morning), but I think we can be even more explicit:

"When an error occurs from a query or mutation, the Apollo Client GQL response distinguishes between two kinds of errors on the `error` object - `graphQLErrors`, an array of errors provided by a GQL server-side resolver, and `networkError`, an error that is thrown outside of a resolver. Within `graphQLErrors`, the Apollo Server can return `AuthenticationError`, `ForbiddenError`, and `UserInputError`s. We don’t need to send these to Sentry on the client-side because all errors aside from `UserInputError` are logged by the `fxa-gql-api` package, but a `networkError` should be logged because it means the query was rejected and no data was returned, such as if Apollo Client failed to connect to the GQL endpoint.

All errors should be handled. As a general rule of thumb, validation errors from `graphQLErrors` should be displayed the user, while `networkError`s should report to Sentry.

[your “when performing a GQL mutation section" with example, but change “Example” to “Example of `useMutation` without user input”]

Example of `useMutation` with user input, checking for `graphQLErrors`:

[example checking for `if (error.graphQLErrors?.length) {`]"


I think we can move the “Testing” part into the “Testing” section too, in other sections we say "See the "Testing" section of this doc for how to X."